ioctl/util/util.go

```diff
@@ -64,6 +76,31 @@ func ConnectToEndpoint(secure bool) (*grpc.ClientConn, error) {
 	return grpc.Dial(endpoint, grpc.WithTransportCredentials(credentials.NewTLS(&tls.Config{})))
 }
 
+// GetAPIClientAndContext establishes a gPRC connection, and return APIServiceClient and context built on the connection
+func GetAPIClientAndContext() (iotexapi.APIServiceClient, context.Context, error) {
+	if gPRCConnInstance == nil {
+		conn, err := ConnectToEndpoint(config.ReadConfig.SecureConnect && !config.Insecure)
+		if err != nil {
+			return nil, nil, output.NewError(output.NetworkError, "failed to connect to endpoint", err)
+		}
+		gPRCConnInstance = conn
+
+		//Automatically close gRPC connection after predefined time
+		timeOut, _ := time.ParseDuration(gRPCConnectionTimeOut)
+		time.AfterFunc(timeOut, func() {
+			conn.Close()
+			gPRCConnInstance = nil
```

Comment:
i think let's not complicate things too much for now, just create a new APIServiceClient and Context and return them

for instance, this code is not thread-safe, if 2 terminals are opened, terminal 1 created the `conn`, terminal 2 got the `conn` right before timeout, while it is still using the connection, timeout fires and the connection is closed.

Reply:
Does this simply implementation work?
```
func GetAPIClientAndContext(conn *grpc.ClientConn) (cli iotexapi.APIServiceClient, ctx context.Context) {
	cli = iotexapi.NewAPIServiceClient(conn)
	ctx = context.Background()
	jwtMD, err := JwtAuth()
	if err == nil {
		ctx = metautils.NiceMD(jwtMD).ToOutgoing(ctx)
	}
}
```

Reply:
The API `Timer.Reset` is used to delay the close of connection each time the func is revoked.